app/build.gradle

```diff
@@ -77,6 +77,14 @@ dependencies {
     implementation 'com.google.dagger:dagger:2.25.3' // for errors
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+
```

Comment:
@enavhasson all of those (maybe without firebase-database) already exist here, so it might cause a bug to do a second import

Reply:
I will check again . I not found that